models.go

```diff
@@ -0,0 +1,35 @@
+package dnsexit
+
+type Action int64
+
+const (
+	Set Action = iota
+	Append
+	Delete
+)
```

Comment:
You can probably (?) get away with un-exporting these (lowercase). But not a dealbreaker, just a suggestion.